packages/trackerless-network/src/logic/StreamEntryPointDiscovery.ts

```diff
@@ -196,8 +197,8 @@ export class StreamEntryPointDiscovery {
             if (this.config.streams.has(streamPartId)) {
                 const stream = this.config.streams.get(streamPartId)
```

Comment:
I would move the non-null assertion `!` here for clarity (we just checked with `has` that it does exist -> no need to go forward with a `stream` of type `StreamObject | unefined`).

```suggestion
                const stream = this.config.streams.get(streamPartId)!
```